docs/source/40_deploying_ADAM.md

```diff
@@ -1,57 +1,106 @@
+```
+cgcloud create-cluster spark -s 2 -t m3.large
+```
+
+Once running, you can ssh to `spark-master` with the command:
+```
+cgcloud ssh spark-master
+```
+
+
+Spark is already installed on the `spark-master` machine and slaves, test it by starting a spark-shell
+```
+spark-shell
+exit()
 ```
 
-If you want to use spot pricing, add `--spot-price` as an option to
-`spark_ec2_launch` (below) and `export MY_SPOT_PRICE=1.399`.
+#### Install ADAM
 
-Export the path to your `spark-ec2` script,
+To use the ADAM application on top of Spark, we need to download and install ADAM on `spark-master`
+From the command line on `spark-master` download a release from:
+https://github.com/bigdatagenomics/adam/releases
 
-```bash
-export SPARK_EC2_SCRIPT="/path/to/spark-0.8.1/ec2/spark-ec2"   # CHANGE ME
+As of this writing, CGCloud supports Spark 1.6.2, not Spark 2.x, so download
+the Spark 1.x Scala2.10 release:
+```
+wget https://repo1.maven.org/maven2/org/bdgenomics/adam/adam-distribution_2.10/0.20.0/adam-distribution_2.10-0.20.0-bin.tar.gz
+tar -xvfz adam-distribution_2.10-0.20.0-bin.tar.gz
 ```
 
-Set up some aliases for commands to the spark ec2 script,
+You can now run `./bin/adam-submit` and `./bin/adam-shell` using your EC2 cluster.
+
+#### Input and Output data on HDFS and S3
+
+Spark requires a file system, such a HDFS or a network file mount, that all machines can access.
+The CGCloud EC2 Spark cluster you just created is already running HDFS.
 
-```bash
-alias spark_ec2_launch="$SPARK_EC2_SCRIPT -k $MY_KEYPAIR \
--i $MY_KEYFILE -s $MY_CLUSTER_SIZE --zone us-east-1c \
---instance-type=$MY_INSTANCE_TYPE launch $MY_CLUSTER_NAME"
-alias spark_ec2_stop="$SPARK_EC2_SCRIPT stop $MY_CLUSTER_NAME"
-alias spark_ec2_start="$SPARK_EC2_SCRIPT -i $MY_KEYFILE start $MY_CLUSTER_NAME"
-alias spark_ec2_destroy="$SPARK_EC2_SCRIPT destroy $MY_CLUSTER_NAME"
-alias spark_ec2_login="$SPARK_EC2_SCRIPT -k $MY_KEYPAIR -i $MY_KEYFILE login $MY_CLUSTER_NAME"
+The typical flow of data to and from your ADAM application on EC2 will be:
+- Upload data to AWS S3
+- Use Conductor (described below) or otherwise transfer from S3 to the HDFS on your cluster
+- Compute with ADAM, write output to HDFS
+- Copy data you wish to persist for later use to S3 using Conductor
+
+For small test files you may wish to skip S3 by  uploading directly to spark-master using `scp` and then copy to HDFS using `hadoop fs -put sample1.bam /datadir/` 
+
+From ADAM shell, or as parameter to ADAM submit, you would refer HDFS URLs  such as:
+```
+adam-submit transform hdfs://spark-master/work_dir/sample1.bam \
+                      hdfs://spark-master/work_dir/sample1.adam
+```
+
+To transfer large amounts of data back and forth from S3, we suggest using [Conductor](https://github.com/BD2KGenomics/conductor).
+
+Its also possible to directly use AWS S3 as a distributed file system, but with some loss of performance.
+( example to be added )
```

Comment:
Nit: I might drop the `example to be added` bit and remove the paragraph break between this paragraph and the conductor paragraph.